<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(255, 255, 255)</string>
  </property>
  <widget class="QFrame" name="under">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>435</y>
     <width>800</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(170, 255, 255)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>80</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>测试</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextBrowserInteraction</set>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>130</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>120</width>
      <height>45</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>380</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid; border-width: 1px; border-radius: 10px; border-color: black;</string>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>400</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>设置起点</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>400</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>设置终点</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>411</width>
     <height>200</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>411</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>441</width>
     <height>200</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#widget
{border-style: solid; border-width: 1px; border-radius: 10px; border-color: black;}</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
